libs/pika/async_mpi/src/mpi_polling.cpp

```diff
@@ -38,8 +38,22 @@
 #include <utility>
 #include <vector>
 
-namespace pika::mpi::experimental {
+#if __has_include(<mpi-ext.h>)
+# include <mpi-ext.h>
+# ifndef OMPI_HAVE_MPI_EXT_CONTINUE
+#  pragma warning "MPIX Continuations not found in <mpi-ext.h>"
+# endif
+
+# define MPIX_RESULT_CHECK(func)                                                                   \
```

Comment:
```suggestion
# define PIKA_MPI_CONTINUATION_RESULT_CHECK(func)                                                                   \
```

Reply:
Actually, does this need to be a macro? `static inline` function might make more sense?